Source/JavaScriptCore/dfg/DFGPhase.h

```diff
@@ -35,9 +35,10 @@ namespace JSC { namespace DFG {
 
 class Phase {
 public:
-    Phase(Graph& graph, const char* name)
+    Phase(Graph& graph, const char* name, const bool validateGraph = true)
```

Comment:
Probably we should rename this flag. When seeing validateGraph is true, I think we will always validate that. But this is not since it also requires validateGraphAtEachPhase. So, I wonder if we have inverted flag instead, disableGraphValidation.